src/unstructured_client/_hooks/custom/request_utils.py

```diff
@@ -50,6 +52,51 @@ def create_request_body(
     return body
 
 
+async def retry_with_backoff_async(
```

Comment:
is it not possible to use the existing `RetryConfig` class defined in `retries.py` instead of creating a new one?

Reply:
Yes, let me switch that back. I copied it over because there was a bug with the autogenerated function (it was using a blocking `time.sleep`, and it didn't have any debug logging. I believe these are both fixed in the latest speakeasy update.